Workshops/CustomConnectorSPN/README.md

```diff
@@ -0,0 +1,484 @@
+
+    ![Add delegated permission User.Invite.All to be able to invite guest users to the organization](./assets/app-registration-add-delegated-permission.png)
+
+1. After adding the permission, you will see that the delegated permission `User.Invite.All` is added to the `Configured permissions` section, but it's not yet granted. This means we will have to grant admin consent, but we will not do this yet, since we also have to add the application permission for the same permission
+1. Select the **Add a permission** button again
+1. In the side panel, select Microsoft Graph
+1. Select Application permissions
+1. In the search box, type `User.Invite`, (1) expand the `User` part, (2) select the checkbox in front of `User.Invite.All` and (3) select the blue **Add permissions** button
+
+    ![Add application permission User.Invite.All to be able to invite guest users to the organization](./assets/app-registration-add-application-permission.png)
+
+1. Take a look at the status of the permissions, it should show that the permissions we just added are not granted yet
+
+    ![Grant admin consent for permissions](./assets/app-registration-grant-consent.png)
+
+1. Select the **Grant admin consent** button
+1. In the confirmation pop up, select the blue **Yes** button
+1. Now, the status should be changed to `Granted`
+
+    ![Permission status changed to granted](./assets/app-registration-granted-consent.png)
+
+1. Select **Certificates & secrets** in the left navigation
+1. In the `Clients secrets` tab, create a new client secret by selecting the **New client secret** button
+
+    ![New client secret](./assets/app-registration-create-client-secret.png)
+
+1. Enter `LabSecret` as the **Description**
+1. Leave **Expires** on the recommended (default) value
+1. Select the blue **Add** button at the bottom
+1. Copy the value of the client secret we just created and save it in a safe place
+
+    ![Copy the value of the client secret](./assets/app-registration-copy-client-secret.png)
+
+## Finish the connector
+
+### Security section
+
+Now we have finished setting up the app registration, we can set our focus on the custom connector again! Let's go back to the browser tab where we left of and continue setting up the custom connector!
+
+1. Select the checkbox in front of **Enable Service Principal support**
+1. In the **Client ID** text box, paste the client ID we copied to notepad
+1. In the **Client Secret** text box, paste the client secret we saved in a safe place at the end of our last task
+1. In the **Resource URL** text box, enter the following URL:
+
+    ```https://graph.microsoft.com```
+
+1. In the **Scope** text box, enter the following permission:
+
+    ```User.Invite.All```
+
+> [!NOTE]
+> The security details (app ID, app secret, etc) in the security section are for OAuth (delegated) authentication, not for the application (SPN) permissions. The SPN details will live in the connection.
```

Comment:
Good comment - it would be great if the image could be marked up too to explain that there are two separate AppID/Secrets - the ones in the connector is for the Delegated OAuth - and the SPN will be filled out later when creating the connector. Normally, these would be two different registrations in Entra.